hack/e2e.sh

```diff
@@ -26,6 +23,22 @@ E2E_KUTTL_TEST=${E2E_KUTTL_TEST:-}
 # Define a custom external network
 export E2E_EXTERNAL_NETWORK_NAME=${E2E_EXTERNAL_NETWORK_NAME:-private}
 
+creds_dir=$(mktemp -d)
+trap 'rm -rf -- "$creds_dir"' EXIT
```

Comment:
We'll need to update the trap now that another one for the same signal merged. I'll take care of it.